City-of-Helsinki/hel-fi-drupal-grants
# AU-819: Robot tests for public website and profile form

# [AU-819](https://helsinkisolutionoffice.atlassian.net/browse/AU-819)

## What was done


These tests focus on public website and profile form.
- Modified the tests and pipeline to use similar variables as in `lomaketyokalu` (like environment, browser type)
- Improved public website tests
- Separated two different tests for application forms
- Fixed Azure devops pipeline setup
- Profile form template needed to have a class added for better selector targeting in tests

## How to test


`make test-robot`, everything should pass
*Except* if you don't have "Yleisavustushakemus" published, then that test (number 5) will fail.


[AU-819]: https://helsinkisolutionoffice.atlassian.net/browse/AU-819?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
